packages/hero/types.ts

```diff
@@ -0,0 +1,25 @@
+import { ActionsProps } from '@hashicorp/react-actions/types'
+import { EyebrowProps } from './eyebrow-with-pattern'
+import { InlineVideoProps } from '@hashicorp/react-inline-video/types'
+import { Products } from '@hashicorp/platform-product-meta'
+
+type WithOptional<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>
+
+export type ContentProps = WithOptional<InlineVideoProps, 'url'> &
```

Comment:
Non-blocking thought, but can we make the prop name for the video be `videoUrl` vs `url`?